pom.xml

```diff
@@ -160,8 +160,8 @@
     </dependency>
     <dependency>
       <groupId>org.seleniumhq.selenium</groupId>
-      <artifactId>selenium-htmlunit-driver</artifactId>
-      <version>${selenium.version}</version>
+      <artifactId>htmlunit-driver</artifactId>
+      <version>2.20</version>
```

Comment:
the htmlunit-driver  was [split out](https://github.com/SeleniumHQ/htmlunit-driver) of the main selenium project and has it's own lifecycle.